### ISSUE TITLE
Model weights after Adaption

### ISSUE THREAD

**Issue by orionw** [Andrew03/transformer-abstractive-summarization#1]
This repo is super great - I hope all goes well with the paper submission!

I was looking to take a model like this and play with it but doing the Adaptive training myself (although you have good instructions) would require 20 epochs of training, if I read the paper correctly.  This wouldn't be too bad except that each epoch takes over 12 hours on the best GPUs I have access to.  I _could_ go through all the work to get the results but it would be expensive and I'd prefer not to.  

Is there anyway you could upload the weights you used for the summarization tasks?  If that's not feasible I would understand but I thought I would ask.

Thanks!

**Comment by atcbosselut** [comment]
Hello,

That should be possible. Is there a particular dataset you're looking for the adapted weights from?

**Comment by orionw** [comment]
Thank you for the response!

The newsroom weights would be awesome if that would be possible.

**Comment by peter-xbs** [comment]
> Hello,
> 
> That should be possible. Is there a particular dataset you're looking for the adapted weights from?

Could you also provide the can_dm weights?

**Comment by CreateRandom** [comment]
Hi,

I'd also be interested in obtaining the weights for cnn_dm. If possible and still available, could you also provide the generated summaries for validation and test set?

**Comment by vprelovac** [comment]
@atcbosselut Are you going to release pre-trained weights for Newsroom? Would make evaluating the model much easier! Thanks!
